### DISCUSSION TITLE
Asking help for LDAP configuration

### DISCUSSION THREAD

**Discussion by cedgnans** [opendcim/openDCIM#1576] [Q&A]
Hello. I'm asking for help please. I've been trying to configure LDAP authentification for my openDCIM.
I've followed the documentation on https://github.com/opendcim/openDCIM/wiki/LDAP and read the video on youtube but I'm not able to do it.
I've thought that it was my openLDAP architecture (tree) but when I even change to reproduce exactly the architecture in the youtube video it didn't work
The worst things it that I don't know where the problem is.
Here is the apache error log

dcim-admin@opendcim:/var/log/apache2$ sudo tail -f error.log
[sudo] password for dcim-admin: 
[Thu Nov 07 09:37:28.490791 2024] [auth_basic:error] [pid 67936] [client 192.168.60.199:59866] AH01618: user noc not found: /
[Thu Nov 07 09:37:45.894407 2024] [auth_basic:error] [pid 67945] [client 192.168.60.199:48682] AH01618: user admin not found: /
[Thu Nov 07 11:05:16.074610 2024] [auth_basic:error] [pid 69916] [client 192.168.60.199:46096] AH01617: user dcim: authentication failure for "/": Password Mismatch
[Thu Nov 07 11:05:30.826310 2024] [auth_basic:error] [pid 67940] [client 192.168.60.199:60828] AH01618: user noc not found: /
[Thu Nov 07 11:08:34.858543 2024] [auth_basic:error] [pid 68788] [client 192.168.60.199:54374] AH01618: user noc not found: /
[Thu Nov 07 11:19:36.798751 2024] [auth_basic:error] [pid 69569] [client 192.168.60.199:35174] AH01617: user dcim: authentication failure for "/": Password Mismatch
[Thu Nov 07 11:21:14.218118 2024] [auth_basic:error] [pid 67939] [client 192.168.60.199:52608] AH01618: user customer not found: /
[Thu Nov 07 14:46:10.040417 2024] [auth_basic:error] [pid 67937] [client 192.168.60.199:42036] AH01617: user dcim: authentication failure for "/": Password Mismatch
[Thu Nov 07 14:46:52.989846 2024] [auth_basic:error] [pid 70408] [client 192.168.60.199:54734] AH01618: user noc not found: /
[Thu Nov 07 14:47:02.690303 2024] [auth_basic:error] [pid 67938] [client 192.168.60.199:43992] AH01618: user noc not found: /

**Comment by samilliken** [comment]
Well your instance is still configured to use auth_basic, which is the password file that you specified, most likely, in .htaccess.   To use LDAP you have to disable the auth_basic.

Scott

________________________________
From: cedgnans ***@***.***>
Sent: Thursday, November 7, 2024 10:58 AM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] Asking help for LDAP configuration (Discussion #1576)


Hello. I'm asking for help please. I've been trying to configure LDAP authentification for my openDCIM.
I've followed the documentation on https://github.com/opendcim/openDCIM/wiki/LDAP and read the video on youtube but I'm not able to do it.
I've thought that it was my openLDAP architecture (tree) but when I even change to reproduce exactly the architecture in the youtube video it didn't work
The worst things it that I don't know where the problem is.
Here is the apache error log

***@***.***:/var/log/apache2$ sudo tail -f error.log
[sudo] password for dcim-admin:
[Thu Nov 07 09:37:28.490791 2024] [auth_basic:error] [pid 67936] [client 192.168.60.199:59866] AH01618: user noc not found: /
[Thu Nov 07 09:37:45.894407 2024] [auth_basic:error] [pid 67945] [client 192.168.60.199:48682] AH01618: user admin not found: /
[Thu Nov 07 11:05:16.074610 2024] [auth_basic:error] [pid 69916] [client 192.168.60.199:46096] AH01617: user dcim: authentication failure for "/": Password Mismatch
[Thu Nov 07 11:05:30.826310 2024] [auth_basic:error] [pid 67940] [client 192.168.60.199:60828] AH01618: user noc not found: /
[Thu Nov 07 11:08:34.858543 2024] [auth_basic:error] [pid 68788] [client 192.168.60.199:54374] AH01618: user noc not found: /
[Thu Nov 07 11:19:36.798751 2024] [auth_basic:error] [pid 69569] [client 192.168.60.199:35174] AH01617: user dcim: authentication failure for "/": Password Mismatch
[Thu Nov 07 11:21:14.218118 2024] [auth_basic:error] [pid 67939] [client 192.168.60.199:52608] AH01618: user customer not found: /
[Thu Nov 07 14:46:10.040417 2024] [auth_basic:error] [pid 67937] [client 192.168.60.199:42036] AH01617: user dcim: authentication failure for "/": Password Mismatch
[Thu Nov 07 14:46:52.989846 2024] [auth_basic:error] [pid 70408] [client 192.168.60.199:54734] AH01618: user noc not found: /
[Thu Nov 07 14:47:02.690303 2024] [auth_basic:error] [pid 67938] [client 192.168.60.199:43992] AH01618: user noc not found: /

**Reply by cedgnans** [reply]
I've disabled the auth_basic

Check it out


![Capture d’écran_2024-11-11_09-42-15](https://github.com/user-attachments/assets/93d45114-a5fe-4775-9c79-8e14b434df43)

**Comment by samilliken** [comment]
Within the apache configuration or just the db.inc.php, because what you showed in your logs says you still have it enabled for apache.

Scott

________________________________

**Reply by cedgnans** [reply]
I don't get it.
However, I've enabled LDAP authentication and disabled basic authentication in the db.inc.php-dist file as shown in the screenshot above.
I can't clearly identify the problem

**Reply by cedgnans** [reply]
![image](https://github.com/user-attachments/assets/138ba58d-071f-4ceb-a0f0-bd96a8d8b96b)

I think the type of error changed
